scripts/build.ts

```diff
@@ -14,14 +26,23 @@ yargs(process.argv.slice(2))
     describe: "Generate the web-features npm package",
     handler: buildPackage,
   })
+  .command({
+    command: "extended-json",
+    describe: "Generate a web-features JSON file with BCD per-key statuses",
+    handler: buildExtendedJSON,
+  })
   .parseSync();
 
 function buildPackage() {
   const packageDir = new URL("./packages/web-features/", rootDir);
   const filesToCopy = ["LICENSE.txt", "types.ts", "schemas/data.schema.json"];
 
+  if (!valid(data)) {
+    logger.error("Data failed schema validation. No package built.");
+    process.exit(1);
+  }
+
   const json = stringify(data);
-  // TODO: Validate the resulting JSON against a schema.
```

Comment:
Since I had to figure out how to validate the built JSON against the schema for the extended JSON, I did the same for the packaged one.